web_widget_image_download/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===========================
```

Comment:
you could update README according to new specs https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst